tests/test_sync_from_manifest.py

```
@@ -0,0 +1,663 @@
+        assert code == EXIT_MANIFEST
+
+    def test_source_error_exit_4(self, tmp_path, target):
+        code = self._run_cli(
+            ["--source", str(tmp_path / "nope"), "--target", str(target)]
+        )
+        assert code == EXIT_SOURCE
+
+    def test_exit_codes_via_real_subprocess(self, source, target, tmp_path):
+        # main() returns the code; __main__ does sys.exit(main()). Prove the
+        # wiring end-to-end for a couple of codes via a real process.
+        drift = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)